tracing-appender/src/non_blocking.rs

```diff
@@ -49,28 +94,37 @@ impl NonBlocking {
         )
     }
 
+    /// Returns a counter for the number of times logs where dropped. This will always return zero if
+    /// `NonBlocking` is not lossy.
     pub fn error_counter(&self) -> Arc<AtomicU64> {
```

Comment:
We do not currently record IO errors that occur when writing logs, but we might want to in the future. Since _this_ counter only counts the number of times we dropped something, I feel like we should rename this (in a separate branch). Maybe `num_dropped` or `dropped_counter` or something.

Reply:
Makes, sense. By seperate branch you mean raise another PR?

Reply:
(Yep, a separate PR.)